WordPress/Tests/CodeAnalysis/EscapedNotTranslatedUnitTest.inc

```diff
@@ -0,0 +1,8 @@
+<?php
+
+esc_html( 'text' );
+esc_html( $var );
+
+esc_html( 'text', 'domain' ); // Warning.
+esc_html( $foo, $bar ); // Warning.
```

Comment:
This should be an error because one should never use a variable for text domain name substitution.

Should it be included in the unit test?

https://pascalbirchler.com/text-domain-wordpress-internationalization/
https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#text-domains

Reply:
No. That is covered by the `I18n` sniff.
A unit test case file should just and only test the sniff it is related to and can contain errors and problems which are forbidden by other sniffs to make sure the _sniff being tested_ handles this correctly.

The principle is that sniffs should function independently of each other. When one sniff is turned off, this shouldn't impact the functioning of any of the other sniffs.

Reply:
Oh, I get it! Thanks for the explanation. This would fall under testing in isolation when doing unit tests, right?

Reply:
Correct :+1: